packages/connectivity/connectivity_macos/README.md

```diff
@@ -6,25 +6,27 @@ The macos implementation of [`connectivity`].
 
 ### Import the package
 
-To use this plugin in your Flutter Web app, simply add it as a dependency in
-your `pubspec.yaml` alongside the base `connectivity` plugin.
 
-_(This is only temporary: in the future we hope to make this package an
-"endorsed" implementation of `connectivity`, so that it is automatically
-included in your Flutter macos app when you depend on `package:connectivity_macos`.)_
+This package has been endorsed, meaning that you only need to add `connectivity`
+as a dependency in your `pubspec.yaml`. It will be automatically included in your app
+when you depend on `package:connectivity`.
 
 This is what the above means to your `pubspec.yaml`:
 
 ```yaml
 ...
 dependencies:
   ...
-  connectivity: ^0.4.6
-  connectivity_macos: ^0.0.1
+  connectivity: ^0.4.8
   ...
 ```
 
-### Use the plugin
+If you wish to use the macos package only, you can add  `connectivity_macos` as a
+dependency:
 
-Once you have the `connectivity_macos` dependency in your pubspec, you should
-be able to use `package:connectivity` as normal.
+```yaml
+...
+dependencies:
+  ...
+  connectivity_macos: ^0.0.2
+  ...
```

Comment:
This is not needed (but if you want to keep it, the version of connectivity_macos should be `^0.1.0`.

We've also added links to the core plugin documentation for people to find instructions there:

* https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore_web/README.md